### PR TITLE
Localize views

### PR DESCRIPTION
**Pull request by shioyama**
Finishes #7

### CODE REVIEW

**Comment by shioyama** [comment]
Payeasy:

latest-screenshot

Bank Transfer:

latest-screenshot

**Comment by shioyama** [comment]
@camelmasa do you want to check this one too? :smile:

**Comment by camelmasa** [comment]
@shioyama I'll do that :fireworks:

File: config/locales/ja.yml [hunk 1]
```diff
@@ -39,6 +41,12 @@ ja:
       daily-yamazaki: *komoju-konbini-instructions-one-code
       seven-eleven: *komoju-konbini-instructions-one-code
     konbini_how_to_pay: "%{convenience}の支払方法"
+    bank_transfer_order_thanks: *komoju-order-thanks
+    bank_transfer_instructions: "銀行振込の支払方法については以下のリンクをご参考ください"
```
Comment (camelmasa) [inline config/locales/ja.yml]: Please add `。` to last :bow:

**Comment by camelmasa** [comment]
@shioyama Thank you for created that :dancers: LGTM :+1:

**Comment by shioyama** [comment]
@camelmasa thanks for reviewing!

**Pull request merged**